### DISCUSSION TITLE
DM rolls are different from player rolls

### DISCUSSION THREAD

**Discussion by Gorvaine** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#102] [Q&A] [answered]
When as a DM I want to help a player rolling something from her pc sheet, the chatmessage does not identify the roll has a PC roll.

It just identifies (Skill rolled) and GM. 

There is not the PC name anywhere in the chatmessage, nor appears the push in the chatmessage.

**Comment by aMediocreDad** [comment]
Hey there. Neither of these are bugs.

The current implementation of the dice roller does not recognize from which actor sheet the dice are rolled. This has never been implemented, but it is something I want to implement in the future when we make the switch to another dicerolling engine.

If you want to interact with the push-button on a chat card as a GM you need to enable "Allow unlimited pushing" in the System settings. See #98 for reference.

Hope this answers your questions.

**Comment by aMediocreDad** [comment, marked as answer]
Just wanted to let you know that this has since my initial answer been reverted for GMs since NPCs SHOULD be allowed to push rolls. Once I can separate out what Actor type rolls I will remove pushing only from monsters instead.
